### PR TITLE
refactor(schema): remove "record.__getitem__" method

### PR DESCRIPTION
**Pull request by Lee-000**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2840y9a 删除 record.__getitem__ 方法](https://app.clickup.com/t/2840y9a)

File: graviti/schema/builtin.py [hunk 1]
```diff
@@ -272,54 +261,18 @@ class record(PortexBuiltinType):  # pylint: disable=invalid-name
...
-        )
-
     """
 
     fields: Fields = param(ptype=PTYPE.Fields)
     nullable: bool = param(False, ptype=PTYPE.Boolean)
 
     def __init__(
         self,
-        fields: Union[Sequence[Union[Field, Tuple[str, PortexType]]], Mapping[str, PortexType]],
```
Comment (linjiX) [inline graviti/schema/builtin.py:297]: remove Field seems irrelevant about your commit message?

**Review by linjiX — APPROVED** [review]
